src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -348,4 +390,78 @@ void getTxInfoToSign_whenTransactionReceiptNotFoundInSubsequentBlock_shouldThrow
             pegoutBtcTransaction
         ));
     }
+
+    @ParameterizedTest
+    @MethodSource("getTxInfoToSignArgProvider")
+    void getTxInfoToSign_whenTransactionReceiptHasPegoutTransactionCreatedEvent_returnOk(
+        int version,
+        byte[] serializedOutpointValues,
+        List<Coin> expectedOutpointValues
+    )
+        throws HSMReleaseCreationInformationException {
+        // Arrange
+        List<LogInfo> logs = new ArrayList<>();
+
+        CallTransaction.Function releaseRequestedEvent = BridgeEvents.RELEASE_REQUESTED.getEvent();
+        byte[] releaseRequestedSignatureTopic = releaseRequestedEvent.encodeSignatureLong();
+        List<DataWord> topics = new ArrayList<>();
+        topics.add(DataWord.valueOf(releaseRequestedSignatureTopic));
+        topics.add(DataWord.valueOf(pegoutCreationRskTx.getHash().getBytes()));
+        topics.add(DataWord.valueOf(pegoutBtcTx.getHash().getBytes()));
+        logs.add(new LogInfo(PrecompiledContracts.BRIDGE_ADDR.getBytes(), topics, null));
+
+        CallTransaction.Function pegoutTransactionCreatedEvent = BridgeEvents.PEGOUT_TRANSACTION_CREATED.getEvent();
+        byte[] pegoutTransactionCreatedSignatureTopic = pegoutTransactionCreatedEvent.encodeSignatureLong();
+        List<DataWord> pegoutTransactionCreatedTopics = new ArrayList<>();
+        pegoutTransactionCreatedTopics.add(DataWord.valueOf(pegoutTransactionCreatedSignatureTopic));
+        pegoutTransactionCreatedTopics.add(DataWord.valueOf(pegoutBtcTx.getHash().getBytes()));
```

Comment:
No big deal maybe, but the real logs at the moment have the following events, in order: `update_collections`, `release_requested ` and `batch_pegout_created`,
When RSKIP428 is active, the list will be, in order:  `update_collections`, `release_requested ` and `batch_pegout_created`, `pegout_transaction_created`.

To be more realistic, do you think we should include all these events in the `logs` list? Maybe have a utility method to create the `logs` with the original events and add the `pegout_transaction_created` event in this test so it is more clear and focused?

I'm thinking about this because the actual code will be handling more events than just these two.

What do you guys think? @nathanieliov , @marcos-iov , @wilmerrootstock , @julia-zack , @apancorb .

Reply:
It seems valid to me.  We should add them to make the test as realistic as possible.

Reply:
By the way, not all pegout creation rsk tx(the one with release_requested) always has a `batch_pegout_created` event; for example, migrations and rejected pegins do not include that log.

Reply:
Sounds like different scenarios to test

Reply:
Done

Reply:
Dome

Reply:
👍🏽